include/diaspora.php

```diff
@@ -701,6 +704,27 @@ public static function handle_from_contact($contact_id, $gcontact_id = 0) {
 	}
 
 	/**
+	 * @brief get a url (scheme://domain.tld/u/user) from a given Diaspora*
+	 * fcontact guid
+	 *
+	 * @param mixed $fcontact_guid Hexadecimal string guid
+	 *
+	 * @return string the contact url or null
+	 */
+	public static function url_from_fcontact_guid($fcontact_guid) {
```

Comment:
Does it need to be a public function? I would name it "url_from_contact_guid" since I'm planning to maybe include the guid into the contact table.